runtime/ops/io.rs

```diff
@@ -111,6 +111,10 @@ pub fn init_stdio(stdio: Stdio) -> Extension {
   let stdio = Rc::new(RefCell::new(Some(stdio)));
 
   Extension::builder()
+    .middleware(|op| match op.name {
+      "op_print" => op_print::decl(),
+      _ => op,
+    })
```

Comment:
This seems fishy - effectively we're always overriding `op_print` provided by `deno_core`. I guess it makes sense though, other consumers/embedders besides CLI might not want to do that (due to missing stdio pipes).

Reply:
Yeah, I thought so too, but it would require pushing all this code down to deno_core.